jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navbar/NavbarNavsTests.java

```diff
@@ -67,45 +81,33 @@ public void getNameNavbarBrandTest() {
         navbarNavWithDropdown.navbarBrand.is().text(textNavbarBrandWithDropdown);
     }
 
-    @Test
-    public void clickNavbarNavWithDisabledLinksTest() throws RuntimeException {
-        for (int i = 1; i < navbarNavWithDisabled.navbarLinks.size(); i++) {
-            navbarNavWithDisabled.navbarLinks.get(i).is().text(containsString(textLinksnavbarNavWithDisabled.get(i)));
-            try {
-                navbarNavWithDisabled.navbarLinks.get(i).click();
-                assertThat(windowsCount(), is(2));
-                switchToWindow(2);
-                assertThat(getUrl(), is(urlLinksnavbarNavWithDisabled.get(i)));
-                closeWindow();
-            } catch (RuntimeException e) {
-                assertThat(e.getMessage(), containsString("is not clickable in any parts"));
-                break;
-            }
+    @Test(dataProvider = "linkNavbarDisabledTest")
+    public void clickNavbarNavWithDisabledLinksTest(int i, String text, String url) {
+        UIElement link = navbarNavWithDisabled.navbarLinks.get(i);
+        link.is().text(text);
+        if (link.isDisabled()) {
```

Comment:
it's bad practice to use if conditions in tests, test should have straight logic, so possibly we should create a few tests instead?